test/tests.js

```diff
@@ -594,7 +633,9 @@ describe('DOM assertions', function() {
     })
 
     describe('text', function() {
-      var subject = parse('<div><span class="blurb">example text</span><p>lorem ipsum</p></div>')
+      beforeEach(function() {
+        subject = parse('<div><span class="blurb">example text</span><p>lorem ipsum</p></div>')
+      });
```

Comment:
Since you built this PR off of the `rendered` text additions, how about add tests here now?